### PR TITLE
[WIP] test: 🚀 New Image Snapshots

### LINKED ISSUE #20866: Volunteer for UI testing ci build
Since we have a lot of component. Modify style file may cause breaking that we need a UI test for each PR.

ref https://www.argos-ci.com/

### PR DESCRIPTION
**Pull request by shaodahong**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
Maybe resolve https://github.com/ant-design/ant-design/issues/20866 ?

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by shaodahong** [comment]
Step:

1. Run local server, like `http://127.0.0.1:5000`, responce a empty html with `antd.css`
2. Use `puppeteer` launch and use `renderToStaticMarkup` parse JSX to html
3. Append html and **screenshot**
4. Expect screenshot image to match

**Comment by shaodahong** [comment]
Output fail base64 Ref: https://github.com/americanexpress/jest-image-snapshot/pull/183#issuecomment-602017663

File: package.json [hunk 1]
```diff
@@ -213,6 +218,7 @@
     "preact-compat": "^3.18.5",
     "prettier": "^2.0.1",
     "pretty-quick": "^2.0.0",
+    "puppeteer": "^2.1.1",
```
Comment (orzyyyy) [inline package.json:221]: How about `puppeteer-core`, or it would download the whole Chromium every install?

**Comment by shaodahong** [comment]
Need work in the main git repository.

**Pull request closed without merging**